edgelet/iotedge/src/check/checks/container_connect_upstream.rs

```diff
@@ -152,16 +152,14 @@ impl ContainerConnectUpstream {
             args.extend(&["--workload_uri", &workload_uri]);
         }
 
-        if &port == "443" {
-            let proxy = settings
-                .agent()
-                .env()
-                .get("https_proxy")
-                .map(std::string::String::as_str);
-            self.proxy = proxy.map(ToOwned::to_owned);
-            if let Some(proxy) = proxy {
-                args.extend(&["--proxy", proxy]);
-            }
+        let proxy = settings
```

Comment:
Don't EA and EH also only use `$https_proxy` for HTTPS connections? For proxying MQTT and AMQP we tell people to switch to the Ws versions, no? If so, not using the proxy for MQTT and AMQP was correct.

Reply:
The IoT Edge Documentation [here](https://docs.microsoft.com/en-us/azure/iot-edge/how-to-configure-proxy-support?view=iotedge-2020-11#:~:text=The%20IoT%20Edge%20runtime%20uses%20AMQP%20by%20default%20to%20talk%20to%20IoT%20Hub.%20Some%20proxy%20servers%20block%20AMQP%20ports.%20If%20that%27s%20the%20case%2C%20then%20you%20also%20need%20to%20configure%20edgeAgent%20to%20use%20AMQP%20over%20WebSocket.%20Uncomment%20the%20UpstreamProtocol%20parameter.) mentions that switch to WS is required in case AMQP ports are blocked and not a mandatory requirement. Maybe @micahl can comment?

In case it is, should we look into completely removing AMQP and MQTT Checks for upstream connection?